sentry.client.config.ts

```diff
@@ -23,5 +23,5 @@ Sentry.init({
 
 	// Setting this option to true will print useful information to the console while you're setting up Sentry.
 	debug: false,
-	enabled: process.env.NEXT_PUBLIC_VERCEL_ENV !== undefined,
+	enabled: true,
```

Comment:
Was this change intended?

Reply:
no, I'm sorry about that.
I reverted the change in 49eb1e4. thanks